src/test/kotlin/org/dashj/platform/dpp/identity/IdentityTest.kt

```diff
@@ -99,19 +98,19 @@ class IdentityTest {
         val json2 = HashMap<String, Any?>()
 
         json2["protocolVersion"] = 0
-        json2["type"] = StateTransition.Types.DATA_CONTRACT_CREATE
+        json2["type"] = StateTransition.Types.DATA_CONTRACT_CREATE.value
 
         val bytes = Cbor.encode(json)
         val bytes2 = Cbor.encode(json2)
```

Comment:
cbor encoding should be using an integer fo the enum, not its string representation.